src/convertToObject.js

```diff
@@ -6,7 +6,24 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const resultObj = sourceString
+    .split(';')
+    .map((itemFromArr) => itemFromArr.trim())
```

Comment:
The variable name 'itemFromArr' is not descriptive and does not reflect the nature of the data it holds. Since you are processing style declarations, a more descriptive name like 'styleDeclaration' would improve readability.